src/test/aztec/base/BridgeTestBase.t.sol

```diff
@@ -8,6 +8,7 @@ import {BridgeTestBase} from "./BridgeTestBase.sol";
 contract BridgeTestBaseTest is BridgeTestBase {
     uint256 private constant VIRTUAL_ASSET_ID_FLAG_SHIFT = 29;
     uint256 private constant VIRTUAL_ASSET_ID_FLAG = 0x20000000; // 2 ** 29
+    uint256 internal constant ADDRESS_MASK = 0x00ffffffffffffffffffffffffffffffffffffffff;
```

Comment:
Why does the mask include `00`?

Reply:
Addressed in a3eb8bce7dd21f52b8de828256e5d19ca167d0a8